src/neuroglancer/image_user_layer.ts

```diff
@@ -168,6 +172,7 @@ export class ImageUserLayer extends Base {
           renderScaleHistogram: this.volumeRenderingRenderScaleHistogram,
           localPosition: this.localPosition,
           channelCoordinateSpace: this.channelCoordinateSpace,
+          shaderSelection: this.volumeRenderingShaderSelection,
```

Comment:
I think I would call this `mode` rather than `shaderSelection` --- while perhaps currently only one shader snippet is swapped, as the feature is refined there will surely be many more differences.